packages/core/src/Stores/ui-store.js

```diff
@@ -97,6 +97,8 @@ export default class UIStore extends BaseStore {
         error_message     : '',
     };
 
+    @observable is_logging_in = false;
```

Comment:
Can we please move this to the Client store?

Reply:
Ok sure. 👍